source/features/filter-pr-by-build-status.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {checkInline} from '../libs/icons';
+import features from '../libs/features';
+
+function init() {
+	const filterAssignee = select('.table-list-header-toggle > details:nth-last-child(2)');
+
+	filterAssignee.before(
```

Comment:
You can merge line 7 and 9 (avoid the variable)

Reply:
Done.